clang/include/clang/Basic/Diagnostic.h

```diff
@@ -9,7 +9,7 @@
 /// \file
 /// Defines the Diagnostic-related interfaces.
 //
-//===----------------------------------------------------------------------===//
+//===----------------------------------------------------------------------===//]
```

Comment:
still unrelated change. I would suggest you to carefully check all the code in the PR

Reply:
done